docs/data/charts/legend/legend.md

```diff
@@ -12,28 +12,35 @@ In chart components, the legend links series with `label` properties and their c
 
 {{"demo": "BasicLegend.js"}}
 
-## Placement
+## Customization
+
+### Position
 
 The legend can either be displayed in a `'column'` or `'row'` layout controlled with the `direction` property.
 
-It can also be moved by a combination of `position: { vertical, horizontal }` properties and the legend offset values.
-The `position` places the legend just next to the drawing area, and offset values move it relative to this base position.
+It can also be moved by the property `position: { vertical, horizontal }` which defines how the legend aligns into the SVG.
 
 - `vertical` can be `'top'`, `'middle'`, or `'bottom'`.
 - `horizontal` can be `'left'`, `'middle'`, or `'right'`.
-- offsets are set with CSS variables `--ChartsLegend-rootOffsetX` and `--ChartsLegend-rootOffsetY`.
+
+From a given `position`, you can add some space with `padding` which can either be a number, or an object `{ top, bottom, left, right }`.
+This `padding` will add space between the SVG borders and the legend.
 
 Defaults are such that the legend is placed on top of the charts.
 
-{{"demo": "LegendCustomizationNoSnap.js", "hideToolbar": true, "bg": "inline"}}
+{{"demo": "LegendPositionNoSnap.js", "hideToolbar": true, "bg": "inline"}}
+
+### Dimensions
+
+Inside the legend, the mark can be customized with `itemMarkWidth` and `itemMarkHeight` which are the width/height in pixels.
+
+You also have access to the `markGap` which is the gap between the mark and its label given in pixels.
+And `itemGap` which is the gap between two legend items.
 
-## Dimensions
+### Label styling
```

Comment:
Would it make sense to have this on the [styling page](https://deploy-preview-10138--material-ui-x.netlify.app/x/react-charts/styling/) since it doesn't apply only to legends?

Reply:
What about keeping a "Label styling" section on all pages that use text. 

For example in axis page it will have similar content explaining how to customize axis labels and ticks label. Saying what's the name of the styling property and how to access it.

In the styling page, we could have an explanation about which properties are supported. For example, the text baseline, the alignement, or the rotation might be slightly different than standard CSS.